admin-dev/themes/new-theme/js/pages/product/product-map.js

```diff
@@ -25,6 +25,11 @@
 
 const combinationListId = '#combination_list';
 const attachmentsBlockId = '#product_specifications_attachments';
+// It does not include "#" so it can be selected by getElementById
+const bulkCombinationModalId = 'bulk-combination-modal';
+const isSelectedCombinationInputClass = 'combination-is-selected';
+const bulkCombinationSelectAllInPageId = 'bulk-select-all-in-page';
+const progressModalId = 'bulk-combination-progress-modal';
```

Comment:
Ok for the others but this one is not included in other strings so it can be simplified and be hardcoded in `bulkProgressModalId` value